app/containers/Bookings/Details/CaseNotes/CaseNotesContainer.js

```diff
@@ -111,7 +124,7 @@ const mapStateToProps = (immutableState, props) => {
 
   const query = {
     ...queryParams,
-    perPage: parseInt(queryParams.perPage, 10) || 10,
+    perPage: parseInt(queryParams.perPage, 10) || 20,
```

Comment:
I remember the perPage count being spread across the solution, might be worth pulling this out into a file to make changes easier going forward.